elb_log_ingestor/main.py

```diff
@@ -66,26 +68,28 @@ def start_server():
         logs_to_be_processed, logs_processed, records, parser_stats
     )
     shipper = elasticsearch_shipper.ElasticsearchShipper(
-        #es_client, records, index_pattern, shipper_stats
-        None, records, index_pattern, shipper_stats
+        es_client, records, index_pattern, shipper_stats
     )
 
     # prepare the ApiEndpoint class for use
-    api_endpoint.ApiEndpoint.parser_stats = parser_stats 
+    api_endpoint.ApiEndpoint.parser_stats = parser_stats
     api_endpoint.ApiEndpoint.shipper_stats = shipper_stats
     api_endpoint.ApiEndpoint.fetcher = fetcher
     api_endpoint.ApiEndpoint.shipper = shipper
 
     server = http.server.HTTPServer(server_address, api_endpoint.ApiEndpoint)
 
-    fetcher_thread = threading.Thread(target=fetcher.run, daemon=True)
+    fetcher_thread = threading.Thread(target=fetcher.run, daemon=False)
```

Comment:
I don't know that it really matters which thread is our daemon - I was shuffling these around when troubleshooting something that turned out to be unrelated, this is how they landed 🤷🏻‍♂️